### DISCUSSION TITLE
SD-XL running in Stability Matrix

### DISCUSSION THREAD

**Discussion by agora-coffee** [LykosAI/StabilityMatrix#96] [Ideas]
First, congratulations for your great work. Makes many things very easy.
Are you working on getting it to work with SD-XL?

**Comment by ionite34** [comment]
It should work currently with any Web UI package that supports it. If you're starting out the Fooocus package provides a simple way of running with the XL detailers.

**Reply by fredm00n** [reply]
Thanks for your amazing work! Would you be able to share high-level steps to get SDXL to work in Stability Matrix? Nothing too detailed, I just need to know where to start looking.

You mentioned the Fooocus package but it looks like an entirely new thing to install so I'm unsure how to proceed: https://github.com/lllyasviel/Fooocus#download

Many thanks in advance for your help
